source/tutorials/write-update-documents.txt

```diff
@@ -0,0 +1,298 @@
+.. _java-rs-write-update-docs:
+
+=================
+Update Documents
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code examples, write, change, edit
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to delete
+documents from a MongoDB collection by performing update operations.
+
+An update operation updates one or more documents in a MongoDB collection.
+You can perform an update operation by using the ``updateOne()`` or
+``updateMany()`` methods.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. 
+
+.. TODO:
+.. Once get started ticket merged in, link the get started tutorial below
+
+.. To learn how to create a
+.. free MongoDB Atlas cluster and load the sample datasets, see the
+.. :ref:`<java-rs-getting-started>` tutorial.
+
+.. TODO: When write insert methods merged in, add subscriber methods section includes within subscriber methods section
+
+.. Subscriber Methods
+.. ~~~~~~~~~~~~~~~~~~~
+
+Update Operations
+-----------------
+
+You can perform update operations in MongoDB by using the following methods:
+
+- ``updateOne()``, which updates *the first document* that matches the search criteria
+- ``updateMany()``, which updates *all documents* that match the search criteria
+
+Each update method requires the following parameters:
+
+- A query filter document, which determines which documents to update. For more
+  information about using query filters, see the :ref:`Filters
+  <java-rs-update-filters>` section.
+
+- An update document, which specifies the update operator (the kind of update to
+  perform) and the fields and values that should change. For more information
```

Comment:
S: since we try to use the present tense whenever possible, I'd remove "should"
```suggestion
  perform) and the fields and values to change. For more information
```